ntoskrnl/mm/ARM3/pagfault.c

```diff
@@ -2262,12 +2262,13 @@ _WARN("Session space stuff is not implemented yet!")
     }
 
     /* Quick check for demand-zero */
-    if (TempPte.u.Long == (MM_READWRITE << MM_PTE_SOFTWARE_PROTECTION_BITS))
+    if ((TempPte.u.Long == (MM_READWRITE << MM_PTE_SOFTWARE_PROTECTION_BITS)) ||
```

Comment:
Somehow, Windows only checks for MM_READWRITE. But I can't explain what they do instead, so LGTM...